samples/places-autocomplete-addressform/src/index.ts

```diff
@@ -15,37 +15,41 @@
  */
 
 // [START maps_places_autocomplete_addressform]
-// This sample uses the Autocomplete widget to help the user select a
+// This sample uses the Places Autocomplete widget to help the user select a
 // place, then it retrieves the address components associated with that
 // place, and then it populates the form fields with those details.
 // This sample requires the Places library. Include the libraries=places
 // parameter when you first load the API. For example:
 // <script
 // src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places">
 
-let placeSearch: google.maps.places.PlacesService;
 let autocomplete: google.maps.places.Autocomplete;
+let address1Field: HTMLInputElement;
+let address2Field: HTMLInputElement;
+let postalField: HTMLInputElement;
 
-const componentForm = {
-  street_number: "short_name",
-  route: "long_name",
+const componentLength = {
```

Comment:
```suggestion
const componentTextAttribute = {
```

This seems like an odd pattern to use a bunch of string literals.

Reply:
refactored